app/helpers/a_z_helper.rb

```diff
@@ -0,0 +1,13 @@
+module AZHelper
```

Comment:
I also think this is a good idea because we might end up using this for other lists across the site. I didn't initially set it up this way because it seemed a bit over-engineered to do a class only to be used once.